src/Sensor_TEST.cc

```diff
@@ -150,6 +150,14 @@ TEST(DOMSensor, MoveAssignment)
   ASSERT_TRUE(nullptr != sensor2.MagnetometerSensor());
   EXPECT_DOUBLE_EQ(mag.XNoise().Mean(),
                    sensor2.MagnetometerSensor()->XNoise().Mean());
+
+  sdf::Magnetometer *magMutable = sensor2.MagnetometerSensor();
```

Comment:
should we add some tests for the other methods?

Reply:
Added in 6f7b832